frontend/src/index.js

```diff
@@ -4,6 +4,7 @@ import "./index.css";
 import App from "./App";
 import * as serviceWorker from "./serviceWorker";
 import { BrowserRouter } from "react-router-dom";
+
```

Comment:
why add a space?

Reply:
i prefer add some space between redux import and other imports to add readability